### ISSUE TITLE
Download function is not working on website?

### ISSUE THREAD

**Issue by LauraWae** [lingdb/Sound-Comparisons#311]
When hitting the button, nothing happens. When hitting again, this error message appears: 
"An error occured when trying to download sound files:
Another download is currently running."

**Comment by runjak** [comment]
Indeed I could replicate this.
![screenshot_2016-03-04_10-33-34](https://cloud.githubusercontent.com/assets/5417642/13523242/9268d2ba-e1f4-11e5-9a65-f31fe0c33792.png)
I'm guessing that some part of creating the `.zip` of the required sound files may not work since the migration to the Jena server.

**Comment by LauraWae** [comment]
Note to myself: Ask Jena people next time I'm in.

**Comment by LauraWae** [comment]
Hi, this issue is still open. Some of the transcribers use the website to download the sound files, so it would be good for them to continue to be able to do so (but also the general public, of course!)

**Comment by runjak** [comment]
Yes, this is correct - I'm a bit pressed for time on this issue at the moment, but will try to make an effort soon.
